python/pipx/Portfile

```diff
@@ -29,7 +29,7 @@ if {
     ![variant_isset python312] &&
     ![variant_isset python313]
 } {
-    default_variants +python312
+    default_variants +python313
```

Comment:
no, the default should stay at 3.22 for now. MacPorts updates its default Python version on January 1st each year, only then ports should start to depend by default on Python 3.13

Reply:
Ok, thanks. Should I close this PR and then resubmit next month? Or, should we leave this PR open until then? Which would be better?

Reply:
you can re-open it next month.